### ISSUE TITLE
NVDA Stops Reading After Opening a File or Folder from Within VSCode, or Closing the Dialog

### ISSUE THREAD

**Issue by thgcode** [microsoft/vscode#326546]
Type: <b>Bug</b>

This is a copy of issue #238520. I couldn't comment on it.
NVDA Version: 2026.1.1. Tested also with NVDA 2025.3.3 and the same behavior happens.

1. Open VSCode.

2. Use the menu or keyboard shortcut to access the open file or open folder options.

3. Close the dialog or open a file or folder.
NVDA says unknown on every interaction afterwards.
If Developer logs is opened with `NVDA+F1` shortcut key, NVDA cannot get the IAccessible2 properties of the object, giving an "unknown interface" message.

Note: if the bug didn't trigger on the first try, perform a second time to trigger it.

NVDA log:

```
Developer info for navigator object:
name: None
role: UNKNOWN
processID: 13524
roleText: None
states: DEFUNCT
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.IAccessible.ia2Web.Ia2Web object at 0x075F5B70>
Python class mro: (<class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: None
location: None
value: None
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
appModule: AppModule(code, appName='code - insiders', processID=13524)
appModule.productName: 'Visual Studio Code - Insiders'
appModule.productVersion: '1.130.0-insider'
appModule.helperLocalBindingHandle: c_long(93027776)
appModule.appArchitecture: 'AMD64'
windowHandle: 2033040
windowClassName: 'Chrome_RenderWidgetHostHWND'
windowControlID: 26
windowStyle: 1445986304
extendedWindowStyle: 32
windowThreadID: 1344
windowText: 'Chrome Legacy Window'
displayText: exception: cannot unpack non-iterable NoneType object
IAccessibleObject: <POINTER(IAccessible2) ptr=0xa30f544 at c3f3cb0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=2033040, objectID=-4, childID=-279
IAccessible accName: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible accRole: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible accState: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible accDescription: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible accValue: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible2 windowHandle: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible2 uniqueID: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible2 role: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible2 states: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible2 attributes: exception: (-2147023179, 'Interface desconhecida.', (None, None, None, 0, None))
IAccessible2 relations: exception: 
```

I also tried these two alternate procedures:

Alternate procedure 1:
1. Run "Code - Insiders.exe" with the `--force-renderer-accessibility` command line switch and perform the main reproduction steps. Nothing changed.

Alternate procedure 2:
1. Use NVDA+N to open the nvda menu, preferences, settings.

2. In the dialog that opens, go to the "Advanced category".

3. Check the "I understand that changing these settings may cause NVDA to function incorrectly." checkbox.

4. Change the option "Use UIA with Microsoft Edge and other 
Chromium based browsers when available" from "Default (Only when necessary)" to "Yes".

5. Perform the main reproduction steps. The bug never occurs.

I also tried to create a sample electron script to trigger the bug but it works correctly, I tried to use the same method call VSCode uses to open the file dialog. On some searches I discovered some possible reasons, one of them is that Chromium could be lowering the accessibility mode on the focus change, so it only exposes basic information, but I don't know how could I debug at this level or see some logs, on the searches it returned nothing about on how to view the accessibility mode lowering logs.

VS Code version: Code - Insiders 1.130.0-insider (5e212606d57c0a17f73bc8b7ae0cc9e14bcfd345, 2026-07-17T16:24:39Z)
OS version: Windows_NT x64 10.0.26200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 5600GT with Radeon Graphics         (12 x 3593)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x1002, DEVICE=0x73ff [AMD Radeon RX 6600], DRIVER_VENDOR=AMD, DRIVER_VERSION=31.0.22017.3004 *ACTIVE*<br>GPU1: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.8875<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.87GB (25.62GB free)|
|Process Argv|. --crash-reporter-id d58e4cc4-0c68-484a-af41-59831ce58e25|
|Screen Reader|yes|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
pythonvspyt551:31249597
nativeloc1:31118317
dwcopilot:31158714
dwoutputs:31242946
copilot_t_ci:31333650
g012b348:31231168
pythonrdcb7:31268811
pythonpcpt1:31399616
6518g693:31302842
6abeh943:31336334
envsactivate1:31349248
editstats-enabled:31346256
cloudbuttont:31366566
3efgi100_wstrepl:31403338
ddidtcf:31398485
cp_cls_t_966_ss:31526232
4je02754:31455664
c3h7c220:31478652
cp_cls_t_1081:31454832
conptydll_true:31485575
e9c30283:31453065
test_treatment2:31471001
46204921:31447328
idci7584:31454084
e3e4d672:31454087
ei9d7968:31462942
f860i809:31458078
hhf17803:31462392
7e187181:31482583
h08i8180:31475367
ddid_t:31478204
hmra_i5g22:31518061
bjc72774_agent_sandbox:31538428
7df3h592:31491241
cp_cls_t_1082:31535311
logging_enabled_new:31490725
jb_cp_cls_t_632:31543129
32d76977:31503652
ha629193:31508444
cdk-lw-on:31524445
f9bf7985:31541111
a1ije391_t:31533808
hgf2d445:31510900
41836166:31537719
jbcp_cls_pctr_t:31531130
cp_intellij_t_nes:31548657
gf14b233:31526830
hide-eh-duplicates:31543293
h561g132:31534507
replace-eh-with-ah:31545853
ihg5j128:31534457
f412h606:31543414
enable_editor_pane_layout:31551200
52258b4d:31547953
fbihi617_lg_t:31548132
c7c27ce7:31551212

```

</details>
